spec/database_connection_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'database_connection'
+
+describe DatabaseConnection do
+  describe '.setup' do
+    it 'establishes a connection to a database via PG' do
+      expect(PG).to receive(:connect).with(dbname: 'chitter_test')
+
+      DatabaseConnection.setup('chitter_test')
```

Comment:
So, typically, what we do is set an environment variable in our spec_helper, and that will be picked up by the database connection setup to connect to the right db.
Check the model spec_helper here: https://github.com/makersacademy/course/blob/main/pills/ruby_web_project_setup_list.md#4-make-capybara-talk-to-sinatra

Otherwise, you whole:
```rb
if ENV['ENVIRONMENT'] == 'test'
  DatabaseConnection.setup('chitter_test')
else
  DatabaseConnection.setup('chitter')
end
```
Is not really useful.

By doing this here, you rely on the fact that this line executes after the line in app.rb.